xayn-ai/src/ranker/context.rs

```diff
@@ -1,5 +1,7 @@
 use crate::{data::document_data::CoiComponent, ranker::util::Id};
 
+/// Ranking context calculation according to
+/// [the spec](https://xainag.atlassian.net/wiki/spaces/M2D/pages/770670607/Production+AI+Workflow#3.2-Context-calculations.)
```

Comment:
```suggestion
/// The context used to calculate a document's score.
```
I like the idea of ​​linking the document, but it describes the old context calculation and that could be a bit confusing.